src/components/textfield/TextField.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { bool, string, func,oneOf,object } from 'prop-types';
+import styles from './textfield.scss';
+import classNames from 'classnames';
+
+
+const TextField = ({value, onChange, onFocus, onBlur, label, placeholder, type="text", multiline, } )=>{
+
+   if(!multiline){
+     return(
+      <div className={classNames(styles.defaultRoot)}>
+        <label className={classNames(styles.defaultLabelStyle)}>{label}</label>
+        <input
+          className={classNames(styles.defaultInputStyle)}
```

Comment:
If you choose to do that you would need to conditionally render the style here.

```jsx
const className = classNames(
    multiline && styles.defaultTextAreaStyle,
    multiline && styles.defaultInputStyle,
);

...
className={className}
...
```

Reply:
or...

```
const className = classNames(
   [styles.defaultTextAreaStyle]: multiline,
   [styles.defaultInputStyle]: !multiline,
);
```

Reply:
I would choose one and make it a standard. I personally don't prefer one rather the other. Let's flip a coin? or defend your proposals :smile: For me the first one looks less procedural. but I don't know if there is any difference between them @crysfel @AndrewMusgrave